pkg/cmd/pipelinerun/describe_test.go

```diff
@@ -1212,3 +1212,122 @@ func TestPipelineRunDescribeV1beta1_taskrun_with_no_status(t *testing.T) {
 	}
 	golden.Assert(t, got, fmt.Sprintf("%s.golden", t.Name()))
 }
+
+func TestPipelineRunDescribe_last(t *testing.T) {
+	clock := clockwork.NewFakeClock()
+
+	trs := []*v1alpha1.TaskRun{
```

Comment:
```suggestion
	trs := []*v1beta1.TaskRun{
```

Reply:
@danielhelfand , `tb.TaskRun( )` returns *v1alpha1.TaskRun not *v1beta1.TaskRun and I checked package  `"github.com/tektoncd/pipeline/test/builder"` does not any function for this to return `*v1beta1.TaskRun`

Reply:
You will need to use structs to create the test data: See example [here](https://github.com/tektoncd/cli/blob/7f56bc9c76faf67dde19bb878ccc12166c0c67f5/pkg/cmd/pipelinerun/describe_test.go#L917).

Reply:
Thanks for that, I have added test for v1beta1 .. will push in sometime.